pkg/reconciler/channel/channel.go

```diff
@@ -188,39 +201,50 @@ func (r *Reconciler) updateStatus(ctx context.Context, desired *v1alpha1.Channel
 }
 
 // reconcileBackingChannel reconciles Channel's 'c' underlying CRD channel.
-func (r *Reconciler) reconcileBackingChannel(ctx context.Context, resourceClient dynamic.ResourceInterface, c *v1alpha1.Channel) (*duckv1alpha1.Channelable, error) {
-	channel, err := resourceClient.Get(c.Name, metav1.GetOptions{})
-	channelable := &duckv1alpha1.Channelable{}
-
+func (r *Reconciler) reconcileBackingChannel(ctx context.Context, channelResourceInterface dynamic.ResourceInterface, c *v1alpha1.Channel, backingChannelObjRef corev1.ObjectReference) (*duckv1alpha1.Channelable, error) {
+	lister, err := r.channelableTracker.ListerFor(backingChannelObjRef)
+	if err != nil {
+		logging.FromContext(ctx).Error(fmt.Sprintf("Error getting lister for Channel: %s/%s", backingChannelObjRef.Namespace, backingChannelObjRef.Name), zap.Error(err))
+		return nil, err
+	}
+	backingChannel, err := lister.ByNamespace(backingChannelObjRef.Namespace).Get(backingChannelObjRef.Name)
 	// If the resource doesn't exist, we'll create it
 	if err != nil {
 		if apierrs.IsNotFound(err) {
-			newChannel, err := resources.NewChannel(c)
+			newBackingChannel, err := resources.NewChannel(c)
 			if err != nil {
 				logging.FromContext(ctx).Error("Failed to create Channel from ChannelTemplate", zap.Any("channelTemplate", c.Spec.ChannelTemplate), zap.Error(err))
 				return nil, err
 			}
-			channel, err = resourceClient.Create(newChannel, metav1.CreateOptions{})
+			logging.FromContext(ctx).Debug(fmt.Sprintf("Creating Channel Object: %+v", newBackingChannel))
+			created, err := channelResourceInterface.Create(newBackingChannel, metav1.CreateOptions{})
+			if err != nil {
+				logging.FromContext(ctx).Error(fmt.Sprintf("Failed to create Channel: %s/%s", backingChannelObjRef.Namespace, backingChannelObjRef.Name), zap.Error(err))
```

Comment:
```suggestion
				logging.FromContext(ctx).Error(fmt.Sprintf("Failed to create backing Channel", zap.Any("backingChannel", newBackingChannel), zap.Error(err))
```

Reply:
done this:

`logging.FromContext(ctx).Error("Failed to create backing Channel", zap.Any("backingChannel", newBackingChannel), zap.Error(err))`